### PR TITLE
[backend] add option to show succeeded logfile

### PR DESCRIPTION
**Pull request by adrianschroeter**
jsc#OBS-40

### CODE REVIEW

**Comment by mlschroe** [comment]
'succeeded' should imply 'last'

**Comment by adrianschroeter** [comment]
okay, adapted

File: docs/api/api/api.txt [hunk 1]
```diff
@@ -1273,6 +1273,7 @@ Parameters:
 
   nostream: do not hang if the build is currently running
   last: show log from last finished build
+  succeeded: show succeeded log, usually used together with last
```
Comment (mlschroe) [inline docs/api/api/api.txt]: Could you please change that to 'show succeeded log, implies last'

**Comment by adrianschroeter** [comment]
On Mittwoch, 13. November 2019, 13:00:23 CET Michael Schroeder wrote:
> mlschroe commented on this pull request.
> 
> > @@ -1273,6 +1273,7 @@ Parameters:
>    nostream: do not hang if the build is currently running
>    last: show log from last finished build
> +  succeeded: show succeeded log, usually used together with last
> 
> Could you please change that to 'show succeeded log, implies last'

renamed it even to lastsucceeded now to make it clear. And to be in
sync with other parameters like "lastbuild"

-- 

Adrian Schroeter <adrian@suse.de>
Build Infrastructure Project Manager

SUSE Software Solutions Germany GmbH,  Maxfeldstr. 5, 90409 Nuernberg, Germany
(HRB 247165, AG München), Geschäftsführer: Felix Imendörffer

**Pull request merged**